lib/pact_broker/client/tasks/publication_task.rb

```diff
@@ -87,6 +92,14 @@ def the_branch
           branch
         end
       end
+
+      def the_consumer_version
```

Comment:
You can delete this check. It's unnecessary.

Reply:
I've made the logic simpler now, let me know what you think